lib/identity_cache/query_api.rb

```diff
@@ -74,6 +74,32 @@ def fetch_multi(*ids)
         records
       end
 
+      def prefetch_associations(associations, records)
+        return if records.empty?
+
+        case associations
+        when nil
```

Comment:
Yup, we supported that previously with the `when nil` in hashify_includes_structure.  I'll add a comment.